internal/models/models.go

```diff
@@ -52,10 +52,17 @@ func (p Publisher) AfterSave(tx *gorm.DB) error {
 	return nil
 }
 
+func (CodeHosting) TableName() string {
+	return "publishers_code_hosting"
+}
+
 type CodeHosting struct {
-	gorm.Model
-	URL         string `json:"url" gorm:"not null"`
-	PublisherID string `json:"publisherId"`
+	ID          string         `json:"id" gorm:"primarykey"`
+	URL         string         `json:"url" gorm:"not null"`
+	PublisherID string         `json:"publisherId"`
+	CreatedAt   time.Time      `json:"createdAt" gorm:"index"`
+	UpdatedAt   time.Time      `json:"updatedAt"`
+	DeletedAt   gorm.DeletedAt `json:"-" gorm:"index"`
```

Comment:
We don't want to show all `CodeHosting`'s field when GETting publishers

```suggestion
	ID          string         `json:"-" gorm:"primarykey"`
	URL         string         `json:"url" gorm:"not null"`
	PublisherID string         `json:"-"`
	CreatedAt   time.Time      `json:"createdAt" gorm:"index"`
	UpdatedAt   time.Time      `json:"updatedAt"`
	DeletedAt   gorm.DeletedAt `json:"-" gorm:"index"`
```

Also, the Open API schema for `codeHosting` doesn't match the struct we return: https://github.com/italia/developers-italia-api/blob/fix/12-publishers-implements-paginations-tests/developers-italia.oas.yaml#L1413

Reply:
It doesn't match for createdAt and UpdatedAt, right?
I've now pushed a change.